### ISSUE TITLE
[469b] Show in script editor somehow offset from bytecode

### ISSUE THREAD

**Issue by SeriousBuggie** [OldUnreal/UnrealTournamentPatches#887]
Since we not able show script line number in error messages like that:
`ScriptWarning: ST_TFemale2Bot CTF-XV-SpireVillageV1.ST_TFemale2Bot0 (Function Botpack.Bot.Retreating.BeginState:0026) Accessed None 'Enemy'`
We can goes in opposite way - show in editor this offset somewhere somehow. I mean 0026 from `BeginState:0026`.

This too can be very helpful in narrow down place of error.

**Comment by stijn-volckaert** [comment]
I agree this would be useful, but this is _very_ hard to do since we don't have line information in UPackage files.

**Comment by SeriousBuggie** [comment]
We can have some button which recompile this code with store info in memory about matches offset to lines, without actual save it into UPackage.

**Comment by SeriousBuggie** [comment]
Resolved by ScriptRays Disassemble:
![img](https://github.com/OldUnreal/UnrealTournamentPatches/assets/70026933/cf0a3476-c15f-4835-9f42-7775de446ec0)
